src/test/java/com/pusher/rest/PusherHttpTest.java

```diff
@@ -33,7 +33,7 @@
 
     private Pusher p;
 
-    @Before
+    @BeforeEach
```

Comment:
Why did you change this from `Before` to `BeforeEach`?

Reply:
Because Before doesn't exist in Junit5 anymore.

Reply:
Ah, I see, it was renamed https://stackoverflow.com/a/45037910/1018290.